package.json

```diff
@@ -20,13 +20,12 @@
     "check-dependency-version-consistency": "dist/bin/check-dependency-version-consistency.js"
   },
   "scripts": {
-    "build": "tsc",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "build": "tsc",
     "lint:docs": "markdownlint '**/*.md'",
     "lint:js": "eslint --cache .",
     "lint:package-json": "npmPkgJsonLint .",
     "lint:package-json-sorting": "sort-package-json --check",
-    "lint:package-json-sorting:fix": "sort-package-json --fix package.json",
```

Comment:
Looks like there is an autofixer but there's no `--fix` option. So you can just remove `--fix`.